src/app/shell/section/group/group-pages/about-school/about-school.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
```

Comment:
fyi: for avoiding using class names you can use semantic layout:


`\<section>
  \<div></div>
\</section>`


section {
  div {
    /**styles**/
  }
}